src/hc-06.hpp

```diff
@@ -0,0 +1,99 @@
+// ==========================================================================
+//
+// File      : hc-06.hpp
+// Part of   : C++ library for using the hc-06 bluetooth module.
+//
+// ==========================================================================
+
+// this file contains Doxygen lines
+/**
+ * @class HC06
+ * @author HU R2D2 2018
+ * @date 13/05/2018
+ * @file hc-06.hpp
+ */
+
+
+#ifndef HC06_H
+#define HC06_H
+
+#include "wrap-hwlib.hpp"
+#include "bluetooth.hpp"
+
+/// \brief
+/// HC-06 bluetooth library
+class HC06 : public bluetooth{
```

Comment:
Clang format hasn't been used, please use clang format to format the source files.